geometry_msgs/msg/TransformStamped.msg

```diff
@@ -1,9 +1,13 @@
 # This expresses a transform from coordinate frame header.frame_id
-# to the coordinate frame child_frame_id
+# to the coordinate frame child_frame_id at the time of header.stamp
 #
 # This message is mostly used by the
-# <a href="http://www.ros.org/wiki/tf">tf</a> package.
+# <a href="http://www.ros.org/wiki/tf2">tf2</a> package.
```

Comment:
@tfoote hmm are we reusing the ROS wiki for ROS 2?

Reply:
Good point updated to use index.ros.org